javascriptv3/example_code/cross-services/transcribe-streaming-app/README.md

```diff
@@ -0,0 +1,469 @@
+* This code has not been tested in all AWS Regions. Some AWS services are available only in specific [Regions](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+* Running this code might result in charges to your AWS account. We recommend you destroy the resources when you are finished. For instructions, see [Destroying the resources](#destroying-the-resources).
+* Running the unit tests might result in charges to your AWS account.
+
+## Create the resources
+You can create the AWS resources required for this cross-service example using either of the following:
+- [The AWS Cloud Development Kit (AWS CDK)](#create-the-resources-using-the-aws-cdk)
+- [The AWS Console](#create-the-resources-using-the-aws-console)
+
+### Create the resources using the AWS CDK
+To run the stack using the AWS CLI:
+
+1. Install and configure the AWS CLI following the instructions in the AWS CLI User Guide.
+
+2. Open the AWS Command Console from the *./transcribe-streaming-app* folder.
+
+3. Run the following command, replacing *STACK_NAME* with a unique name for the stack.
+```
+aws cloudformation create-stack --stack-name STACK_NAME --template-body file://setup.yaml --capabilities CAPABILITY_IAM
+```
+**Important**: The stack name must be unique within an AWS Region and AWS account. You can specify up to 128 characters, and numbers and hyphens are allowed.
+
+4. Open [AWS CloudFormation in the AWS Console](https://aws.amazon.com/cloudformation/), and open the **Stacks** page.
+
+![ ](images/cloud_formation_stacks.png)
+
+5. Choose the **Resources** tab. The **Physical ID** of the **IDENTITY_POOL_ID** you require for this cross-service examples is displayed.
+
+![ ](images/cloud_formation_resources_tab.png)
+
+For more information on the create-stack command parameters, see the [AWS CLI Command Reference guide](https://docs.aws.amazon.com/cli/latest/reference/cloudformation/create-stack.html), and the [AWS CloudFormation User Guide](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-cli-creating-stack.html).
+
+### Create the resources using the AWS Console
+####Create an unauthenticated user role
+4. Open [AWS Cognito in the AWS Console](https://aws.amazon.com/cloudformation/), and open the *Stacks* page.
+5. Choose **Manage Identity Pools**.
+6. Choose **Create new identity pool**.
+7. In the **Identity pool name** field, give your identity pool a name.
+7. Select the **Enable access to unauthenticated identities** checkbox.
+8. Choose **Create Pool**.
+9. Choose **Allow**.
+10. Take note of the **Identity pool ID**, which is highlighted in red in the **Get AWS Credentials** section.
+
+![ ](images/identity_pool_ids.png)
+
+11.Choose **Edit identity pool**.
+12. Take note of the name of the role in the **Unauthenticated role** field.
+
+####Adding permissions to an unauthenticated user role
+13. Open [AWS IAM in the AWS Console](https://aws.amazon.com/iam/), and open the *Roles* page.
```

Comment:
AWS IAM -> IAM (subsequent use is just IAM)
AWS **Management** Console

Reply:
fixed